src/main/resources/schema.sql

```diff
@@ -15,3 +15,51 @@ CREATE TABLE IF NOT EXISTS aircraft (
     routing_range numeric NOT NULL,
     name varchar(25)
 );
+
+CREATE TABLE IF NOT EXISTS airport(
+    airport_id char(3) PRIMARY KEY,
+    a_name varchar(25) NOT NULL,
+    a_city varchar(25) NOT NULL,
+    a_country varchar(25) NOT NULL,
+    flight_number varchar(6) NOT NULL
+);
+
+CREATE TYPE wifi_enum AS ENUM ('paid' , 'free', 'no');
```

Comment:
Can you add something like this in `DROP TYPE wifi_enum CASCADE;`. An error will happen stating that the type already exists if you build it again after creating the type. To fix this just add in that line above the creation of the type.